### ISSUE TITLE
failed: SplitBrainResolverIntegrationSpec

### ISSUE THREAD

**Issue by patriknw** [akka/akka#29126]
https://jenkins.akka.io:8498/job/akka-nightly-jdk8/7694

```
[info] [JVM-3] - must KeepSide1 when using keep-oldest and side1=3 and side2=3 (on node 'node3', class akka.cluster.sbr.SplitBrainResolverIntegrationSpecMultiJvmNode3) *** FAILED ***
[info] [JVM-3]   java.lang.AssertionError: assertion failed: timeout (2 seconds) during expectMsg while waiting for 0
[info] [JVM-3]   at scala.Predef$.assert(Predef.scala:223)
[info] [JVM-3]   at akka.testkit.TestKitBase.expectMsg_internal(TestKit.scala:459)
[info] [JVM-3]   at akka.testkit.TestKitBase.expectMsg(TestKit.scala:445)
[info] [JVM-3]   at akka.testkit.TestKitBase.expectMsg$(TestKit.scala:445)
[info] [JVM-3]   at akka.testkit.TestKit.expectMsg(TestKit.scala:969)
[info] [JVM-3]   at akka.cluster.sbr.SplitBrainResolverIntegrationSpec$DisposableSys.$anonfun$verify$24(SplitBrainResolverIntegrationSpec.scala:311)
[info] [JVM-3]   at scala.runtime.java8.JFunction1$mcII$sp.apply(JFunction1$mcII$sp.java:23)
[info] [JVM-3]   at scala.collection.immutable.Range.foreach(Range.scala:158)
[info] [JVM-3]   at akka.cluster.sbr.SplitBrainResolverIntegrationSpec$DisposableSys.$anonfun$verify$23(SplitBrainResolverIntegrationSpec.scala:309)
[info] [JVM-3]   at akka.testkit.TestKitBase.poll$2(TestKit.scala:330)
[info] [JVM-3]   ...

```

**Comment by patriknw** [comment]
Strange. The SBR decision is correct and the sharded entities are started as expected but the expected replies in the awaitAssert is not happening. It has plenty of time remaining.

Attached log with some comments.

[issue-29126.log](https://github.com/akka/akka/files/4681594/issue-29126.log)

**Comment by patriknw** [comment]
I'm pretty sure that this is the same underlying issue as in https://github.com/akka/akka/issues/29131

Keeping it open until we have merged the fixes, since it will probably continue failing until then.

**Comment by patriknw** [comment]
Should be fixed by #29131
